x-pack/plugins/observability_solution/inventory/public/components/grouped_inventory/inventory_group_accordion.tsx

```diff
@@ -29,7 +30,7 @@ export function InventoryGroupAccordion({
   isLoading,
 }: InventoryGroupAccordionProps) {
   const { euiTheme } = useEuiTheme();
-  const field = group[groupBy];
+  const field = flattenObject(group)[groupBy];
```

Comment:
`groupBy` here will be `entity.type`, so we need to flatten the object to be able to access the attribute's value.